test/mysql_change_user_tests.erl

```diff
@@ -0,0 +1,153 @@
+%% MySQL/OTP – MySQL client library for Erlang/OTP
+%% Copyright (C) 2014 Viktor Söderqvist
```

Comment:
Name and year are slightly incorrect :-)

Reply:
I was assuming the copyright year and author name refers to MySQL/OTP itself, not the module in question. I can change it if you want, of course.

Reply:
Normally these things are per file.

The author of each piece of the code owns the copyright, so authors who add code (excluding minor changes) should be added here. I don't mind having my name here, but it's in your interest to add you name here. If I own all the code, hypothetically, I can re-license it however I want, which may not be in your interest. (I don't know how important these lines are though, because if you really want to know exactly who did what and when, you use git blame.)

Reply:
Ok, changed it :)